src/regolith/exemplars.json

```diff
@@ -2175,6 +2175,81 @@
         "email": "haperson@uni.edu",
         "university_id": "HAP42"
     },
+    "software": [
+        {
+            "_id": "diffpy.utils",
+            "groups": ["xrd"],
+            "active": true,
+            "url": "https://github.com/diffpy/diffpy.utils",
+            "grants": ["NSF Funding"],
+            "author": ["Simon Billinge", "Sanjoon Bob Lee", "Zhiming Xu", "Tieqiong Zhang"],
+            "release": [
+                {
+                    "major": 3,
+                    "minor": 1,
+                    "patch": 0,
+                    "release_date": "2025-10-25",
+                    "summary": "Python 3.14 and something else",
+                    "change": ["deprecated python2 feature.",
+                                "changed to scikit-packaged standard.",
+                                "Add some functionality.",
+                                "Modify existing function to make convenient."
+                    ],
+                    "version_number": "1.0.0",
```

Comment:
We don't need this because the version number can be built from major. Minor. Patch